test/rulesets/Base/base.jl

```diff
@@ -72,31 +91,11 @@
             test_scalar(exp2, x)
             test_scalar(exp10, x)
 
-            x isa Real && test_scalar(cbrt, x)
-            if (x isa Real && x >= 0) || x isa Complex
-                # this check is needed because these have discontinuities between
-                # `-10 + im*eps()` and `-10 - im*eps()`
-                should_test_wirtinger = imag(x) != 0 && real(x) < 0
-                test_scalar(sqrt, x; test_wirtinger=should_test_wirtinger)
-                test_scalar(log, x; test_wirtinger=should_test_wirtinger)
-                test_scalar(log2, x; test_wirtinger=should_test_wirtinger)
-                test_scalar(log10, x; test_wirtinger=should_test_wirtinger)
-                test_scalar(log1p, x; test_wirtinger=should_test_wirtinger)
-            end
-        end
-    end
-
-    @testset "Unary complex functions" begin
-        for x in (-4.1, 6.4, 1.0+0.5im, -10.0+1.5im)
-            test_scalar(real, x)
-            test_scalar(imag, x)
-
-            test_scalar(abs, x)
-            test_scalar(hypot, x)
-
-            test_scalar(angle, x)
-            test_scalar(abs2, x)
```

Comment:
These shouldn't have been deleted,
only the cases where they are not holomorphic should have been deleted.

We definately still need tests of `abs` and `hypot` for real values.